book/chapters/chapter1/introduction_and_overview.qmd

```diff
@@ -220,9 +221,9 @@ You will learn how to do all this and more in this book.
 
 ## The `mlr3` Ecosystem
 
-Throughout this book, we interchangeably refer to the package `r mlr3` and the ecosystem `mlr3`, which also encompasses all extension packages.
-The core functionality to all packages in the ecosystem is provided by `r mlr3` but in almost all use cases users will use more packages in the ecosystem, and usually at least the packages in `r mlr3verse`.
-@fig-mlr3verse shows the packages in the `mlr3verse` that extend `r mlr3` with capabilities for preprocessing, pipelining, visualizations, additional learners, additional task types, and more.
+Throughout this book, we often refer to `mlr3`, which does not refer to the single `r mlr3` base package but all the packages in our ecosystem.
+The `r mlr3` *package* provides the base functionality that the rest of the ecosystem depends on for building more advanced machine learning tools.
+@fig-mlr3verse shows the packages in the `r mlr3verse` that extend `r mlr3` with capabilities for preprocessing, pipelining, visualizations, additional learners, additional task types, and much more.
```

Comment:
```suggestion
@fig-mlr3verse shows the packages in our ecosystem that extend `r mlr3` with capabilities for preprocessing, pipelining, visualizations, additional learners, additional task types, and much more.
```

The figure isn't {mlr3verse} but many more packages. We should avoid ever using mlr3verse unless we're talking about the package for installation